igcollect/linux_disklatency.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+"""igcollect - Linux Disk I/O latency
+
+Copyright (c) 2019 InnoGames GmbH
+"""
+
+from argparse import ArgumentParser
+from subprocess import check_output
+from time import time
+import sys
+from libvirt import (
+    open,
+    libvirtError,
+)
+
+def parse_args():
+
+    parser = ArgumentParser()
+
+    parser.add_argument(
+        '--prefix', dest='prefix', type=str, default='None',
+        help='Prefix to Graphite')
+
+    parser.add_argument(
+        '--storagepool', dest='storagepool', type=str, default='None',
+        help='Storage Pool')
+
+    parser.add_argument(
+        '--storagevol', dest='storagevol', type=str, default='None',
+        help='Storage Volume')
+
+    return parser.parse_args()
+
+def main():
+    URI = 'qemu:///system'
+    args = parse_args()
+    template = args.prefix + '.{}.{} {} ' + str(int(time()))
+
+    try:
+        conn = open(URI)
```

Comment:
After changing the import, change this to `libvirt_open('qemu:///system')`. You might as well loose the URI variable, if you only use it here anyway.